src/test/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileValidationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import static org.junit.Assert.*;
+
+import java.sql.Date;
+import java.util.Calendar;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import org.kuali.kfs.krad.exception.ValidationException;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractFile;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractRequestDetailFileLine;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractRequestEntryDetailFileLine;
+
+public class ConcurRequestExtractFileValidationServiceImplTest {
+    private ConcurRequestExtractFileValidationServiceImpl concurRequestExtractFileValidationService;
+    
+    @Before
+    public void setUp() throws Exception {
+        Logger.getLogger(ConcurRequestExtractFileValidationServiceImpl.class).setLevel(Level.DEBUG);
+        concurRequestExtractFileValidationService = new ConcurRequestExtractFileValidationServiceImpl();
+    }
+    
+    @After
+    public void tearDown() throws Exception {
+        concurRequestExtractFileValidationService = null;
+    }
+    
+    @Test
+    public void testHeaderAmountMatches() {
+        ConcurRequestExtractFile testFile = buildConcurRequestExtractFile();
+        if (concurRequestExtractFileValidationService.fileApprovedAmountsMatchHeaderApprovedAmount(testFile)) {
+            assertTrue("Valid: Header amount matched sum of row amounts from file.", true);
+        }
+        else {
+            assertTrue("Invalid: Header amount did NOT match sum of row amounts from file.", false);
+        } 
+    }
+    
+    @Test
+    public void testHeaderAmountDoesNotMatch() {
```

Comment:
I think you need to separate tests here, one for a test case where the amounts match and one for a test case where the amounts don't match.